bridge/bridge.go

```diff
@@ -133,6 +137,16 @@ func (b *Bridge) Run(ctx context.Context) error {
 			if err := b.updateHeader(bot, mmChannel, mapping); err != nil {
 				return err
 			}
+			altChannels := map[string]*model.Channel{}
+			if mapping.Diversions != nil {
```

Comment:
The `if` is unnecssary; iterating over an empty map is defined.

Reply:
Thank you; I though that might be the case, but I wasn't quite confident enough to omit it. Fixed.